docs/extras/tactic_writing.md

```diff
@@ -475,3 +475,23 @@ reading available tactics.
 * `ma <|> ma'` recover from failure: runs `ma` and if it fails then runs `ma'`.
 * `a $> mb`: same as `do mb, return a`
 * `ma <$ b`: same as `do ma, return b`
+
+## Cheap and dirty tactic trick
+
+If you just want one tactic to implement a few lines of tactic mode in Lean (e.g., because you are
+finding that you are having to type the same three or four lines several times in your proof)
+you can hack together a tactic like this:
+
+```lean
+meta def rw_search_hack : tactic unit := do
+`[
+repeat {rw assoc},
+try {simp},
+repeat {rw comp_right_app},
+repeat {rw nat_trans.naturality}
+]
+```
+
+This creates a new tactic which just runs the tactics used in their definition. If you want to use such a tactic such as this in other files, you should put it in the `tactic.interactive` namespace.
```

Comment:
```suggestion
This creates a new tactic which just runs the tactic script between `` `[ `` and `]`.
```
I take back my earlier suggestion. If there's no argument parsing going on, and the name isn't duplicated in `tactic.interactive`, it's okay to put the tactic in another namespace. (But this is unrelated to using the tactic in other files.)